app/src/main/res/values/strings.xml

```diff
@@ -389,6 +389,11 @@
     <string name="local_playlists">Local playlists</string>
     <string name="not_enabled">Menu item not enabled!</string>
     <string name="select_other_start_tab">Please select an other start tab first!</string>
+    <string name="brightness">Brightness</string>
+    <string name="volume">Volume</string>
+    <string name="auto">Auto</string>
+    <string name="swipe_control">Swipe control</string>
```

Comment:
I'd prefer `Swipe controls` as the English string here as your PR adds support for brightness as well as volume, we should change the string identifier name accordingly as well (`swipe_control` -> `swipe_controls`), same for the summary resource name.

Reply:
My bad (English) again :P